### ISSUE TITLE
--keep-direct-and-as options should be removed

### ISSUE THREAD

**Issue by ucodery** [timothycrosley/isort#1192]
This setting now defaults to True, so there is no point in setting it in the shell. It should be marked deprecated and then removed after a few minor releases.

Additionally, there is no mirror to this option (--discard-direct-with-as if you will), and it is not listed as a configurable setting in the wiki (although the tests suggest that it would work if added to a config). The only sure way to get the old behavior is to use the api directly. For these reasons I would suggest also removing keep_direct_and_as_imports from isort code as a whole. IMHO this setting just has the ability to introduce bugs. The following simple script would break if keep_direct_and_as_imports was set to False, for instance in an upstream commit hook.
```python
from os import path
import os.path as pth

def foo(bar):
    return path.pathsep.join(bar)
```

**Comment by timothycrosley** [comment]
Thanks for reporting! This is fixed in develop and slated to be released in the 5.1.0 minor release by the end of the month

**Comment by timothycrosley** [comment]
A fix for this has been deployed to PyPI in the [5.1.0 release](https://timothycrosley.github.io/isort/CHANGELOG/#510-july-14-2020), 

Thanks for reporting!

~Timothy
